src/DRAMPower/DRAMPower/memspec/MemSpecDDR5.h

```diff
@@ -44,6 +44,14 @@ namespace DRAMPower {
 
         double vddq;
 
+        enum class BusConfig {
```

Comment:
Macht es Sinn die Definition einmal für alle Standards in der MemSpec.h zu machen?